app/portainer/views/support/product/product.html

```diff
@@ -47,13 +47,25 @@
               <div style="margin-top: 10px; margin-bottom: 95px;">
                 <label for="endpoint_count" class="col-sm-7 control-label text-left" style="margin-top: 7px;">Hosts</label>
                 <div class="col-sm-5">
-                  <input type="number" class="form-control" ng-model="formValues.hostCount" id="endpoint_count" placeholder="10" min="10">
+                  <input type="number" class="form-control" ng-model="formValues.hostCount" id="endpoint_count"
+                    placeholder="1" min="1" ng-if="product.Id == 1">
+                  <input type="number" class="form-control" ng-model="formValues.hostCount" id="endpoint_count"
+                    placeholder="4" min="4" ng-if="product.Id == 2 || product.Id == 3">
+                  <input type="number" class="form-control" ng-model="formValues.hostCount" id="endpoint_count"
+                    placeholder="10" min="10" ng-if="product.Id == 4 || product.Id == 5">
```

Comment:
Refactor to use a single `input` field.
Bind `placeholder` and `min` to controller state values (define these values onInit based on product Id)